templates/repo/diff/whitespace_dropdown.tmpl

```diff
@@ -3,20 +3,20 @@
 	{{svg "octicon-triangle-down" 14 "dropdown icon"}}
 	<div class="menu">
 		<a class="item" href="?style={{if .IsSplitStyle}}split{{else}}unified{{end}}&whitespace=show-all">
-			<i class="circle {{if eq .WhitespaceBehavior "show-all"}}dot{{else}}outline{{end}} icon"></i>
-			{{.locale.Tr "repo.diff.whitespace_show_everything"}}
+			<input id="whitespace-show-all" class="mr-3 pointer-events-none" type="radio"{{if eq .WhitespaceBehavior "show-all"}} checked{{end}}>
+			<label for="whitespace-show-all" class="pointer-events-none">{{.locale.Tr "repo.diff.whitespace_show_everything"}}</label>
```

Comment:
I think these `id`/`for` are unnecessary. Because the input doesn't respond the input event.

Reply:
They might be useful to screen readers.

Reply:
How about simplified `<label><input ></label>`? Is this style friendly to  screen readers?

Reply:
From [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label):

> The label text is not only visually associated with its corresponding text input; it is programmatically associated with it too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.

Reply:
Nesting might work, need to investigate it.

Reply:
Added nesting, output rendering appears to be visuall unchanged with it.

<img width="300" alt="image" src="https://user-images.githubusercontent.com/115237/192144397-b641513f-2b9a-4ba4-95d5-7982ba6f8723.png">

Reply:
https://jsfiddle.net/3bxjqo86/2/

I have tested, I see no difference

Reply:
Yeah, there should be no issue with nesting and it saves is from declaring IDs. I was just worried about the nesting causing different rendering, but it appears to not be the case.